LaTeX/syntax_test_latex.tex

```diff
@@ -382,6 +390,27 @@ \subsubsection{name}
 %                    ^ source.python.embedded
 
 
+% PACKAGE: array
+% The array package extends array and tabular environments
+% and allows for macros in format specifications
+
+\begin{tabular}[t]{|x|@{See: }>{$}l<{$}|}
```

Comment:
Would be good to add
``` tex
% ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ meta.environment.tabular.latex - meta.environment.tabular.latex meta.environment.tabular.latex
```
to avoid duplicates scopes and ensure the scope is over the whole command.